jenkins_pipelines/manager_testsuite/Jenkinsfile_qa_testsuite

```diff
@@ -41,26 +41,32 @@ pipeline {
                     } else {
```

Comment:
I would say we need to remove all the snapshots from all stages from previous Jenkins run before deploy.
Otherwise, it could fail during `terraform apply` or other not wanted scenarios.

Reply:
Now added a cleanup at the end of the pipeline, don't think it's possible in the beginning of the pipeline to do cleanup of the previous.